kin/blockchain/channel_manager.py

```diff
@@ -32,10 +32,11 @@ def __init__(self, secret_key, channel_keys, network, horizon):
         self.low_balance_builders = []
         for channel_key in channel_keys:
             # create a channel transaction builder.
-            builder = Builder(secret=channel_key, network=network, horizon=horizon)
+            # fee gets updated once a transaction is being built
+            builder = Builder(secret=channel_key, network=network, horizon=horizon, fee=0.01)
```

Comment:
why the builder needs the fee?

Reply:
If you wish to build a transaction manually (with all the obfuscation like in "send_kin"), you need to use the builder, in kin_base sdk the builder holds a single transaction so it holds the fee as well